test/common/markdown/package-lock.json

```diff
@@ -978,6 +998,17 @@
       "requires": {
         "graceful-fs": "^4.1.11",
         "through2": "^2.0.3"
```

Comment:
should this be `^2.0.5`?

Reply:
Not sure, but my bot did it and it looks like it works.

Reply:
Not blocking